js_modules/dagster-ui/packages/ui-components/src/components/__stories__/ListItem.stories.tsx

```diff
@@ -411,3 +411,24 @@ export const SmallList = () => {
     </div>
   );
 };
+
+export const BigPadding = () => {
+  return (
+    <div style={{width: '400px'}}>
+      <ListItem
+        index={0}
+        padding={{vertical: 48, horizontal: 48}}
+        renderLink={renderLink}
+        href="/jobs/foo"
+        left={<div>Left</div>}
+        right={
+          <HorizontalControls
+            controls={[
+              {key: 'menu', control: <Button intent="none" icon={<Icon name="more_horiz" />} />},
+            ]}
+          />
+        }
+      />
+    </div>
```

Comment:
I don't think this would repro the bug

Reply:
Yeah no, this was just for testing the padding issues you were running into.

Reply:
ah gotcha